docker_cli/_exceptions.py

```diff
@@ -0,0 +1,38 @@
+class CommandNotFoundError(Exception):
+    def __init__(self, command_name: str):
+        self.command_name = command_name
+        self.message = (f"Command '{self.command_name}' was run, but is not "
+                        "present on the Docker image.")
+
+    def __str__(self):
+        return self.message
```

Comment:
We should be calling the base class `__init__()` method here. 

If we do that, we don't need to define any methods like `__str__()` -- the base `Exception` class will take care of all the mechanics for us.

```suggestion
        super().__init__(f"Command '{self.command_name}' was run, but is not "
                          "present on the Docker image.")
```

Reply:
(same goes for the other custom exceptions defined here)